File: src/main/java/bot/commands/PlanCommand.java
package bot.commands;

import bot.run.AbitVTBot;
import org.telegram.telegrambots.meta.api.methods.send.SendDocument;
import org.telegram.telegrambots.meta.api.objects.InputFile;
import org.telegram.telegrambots.meta.api.objects.Message;
import org.telegram.telegrambots.meta.api.objects.Update;
import org.telegram.telegrambots.meta.api.objects.replykeyboard.InlineKeyboardMarkup;
import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.InlineKeyboardButton;
import org.telegram.telegrambots.meta.exceptions.TelegramApiException;

import java.util.ArrayList;
import java.util.List;
import java.util.ResourceBundle;

public class PlanCommand implements Command {

    private String getTEXT(String chatId){
        ResourceBundle otherBundle = ResourceBundle.getBundle("other", bot.getSql().selectLanguage(chatId).getLocale());
        return otherBundle.getString("plan.text");
    }

    private final AbitVTBot bot;

    public PlanCommand(AbitVTBot bot) {
        this.bot = bot;
    }

    @Override
    public void execute(Update update) throws TelegramApiException {

        String chatId;

        Message message = update.getMessage();
        SendDocument sendDocument = new SendDocument();
        if (message != null) chatId = message.getChatId().toString();
        else chatId = update.getCallbackQuery().getMessage().getChatId().toString();
        sendDocument.setChatId(chatId);

        InputFile plans = new InputFile();
        plans.setMedia("https://eduold-prod.itmo.dev/file/subspec/4285/09.03.01_kompyuternye_sistemy_i_tehnologii_14549.pdf");
        sendDocument.setDocument(plans);

        sendDocument.setAllowSendingWithoutReply(true);
        bot.execute(sendDocument);

        plans.setMedia("https://eduold-prod.itmo.dev/file/subspec/4290/09.03.04_44.03.04_kompyuternye_tehnologii_v_dizayne_14546_14547.pdf");

        List<List<InlineKeyboardButton>> buttons = new ArrayList<>();
        InlineKeyboardButton button = new InlineKeyboardButton();
        button.setText(getTEXT(chatId));
        button.setUrl("https://docs.google.com/spreadsheets/d/1NlrnPsPksHXzEHFnSHtUtbJg5AENXx6pVFBevHIg-4k/edit#gid=1824976275");

        List<InlineKeyboardButton> bl = new ArrayList<>();
        bl.add(button);
        buttons.add(bl);

        InlineKeyboardMarkup keyboard = new InlineKeyboardMarkup(buttons);

        sendDocument.setReplyMarkup(keyboard);

        bot.execute(sendDocument);
    }

    @Override
    public String help(String chatId) {
        ResourceBundle helpBundle = ResourceBundle.getBundle("help", bot.getSql().selectLanguage(chatId).getLocale());
        return helpBundle.getString("plan");
    }

    @Override
    public String name() {
        return "/plan";
    }

}

File: src/main/java/bot/commands/PlanCommand.kt
package bot.commands

import bot.run.AbitVTBot
import bot.util.CommandsUtil.getChatId
import org.telegram.telegrambots.meta.api.methods.send.SendDocument
import org.telegram.telegrambots.meta.api.objects.InputFile
import org.telegram.telegrambots.meta.api.objects.Update
import org.telegram.telegrambots.meta.api.objects.replykeyboard.InlineKeyboardMarkup
import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.InlineKeyboardButton
import java.util.*

class PlanCommand(private val bot : AbitVTBot) : Command {
    override fun execute(update: Update) {
        val chatId = getChatId(update)

        val sendDocument = SendDocument()
        sendDocument.chatId = chatId

        val plans = InputFile()
        plans.setMedia("https://eduold-prod.itmo.dev/file/subspec/4285/09.03.01_kompyuternye_sistemy_i_tehnologii_14549.pdf")
        sendDocument.document = plans

        sendDocument.allowSendingWithoutReply = true
        bot.execute(sendDocument)

        plans.setMedia("https://eduold-prod.itmo.dev/file/subspec/4290/09.03.04_44.03.04_kompyuternye_tehnologii_v_dizayne_14546_14547.pdf")

        val buttons: MutableList<List<InlineKeyboardButton>> = ArrayList()
        val button = InlineKeyboardButton()
        button.text = bot.resourceOperator.getText("plan.text", chatId)!!
        button.url = "https://docs.google.com/spreadsheets/d/1NlrnPsPksHXzEHFnSHtUtbJg5AENXx6pVFBevHIg-4k/edit#gid=1824976275"

        val bl: MutableList<InlineKeyboardButton> = ArrayList()
        bl.add(button)
        buttons.add(bl)

        val keyboard = InlineKeyboardMarkup(buttons)

        sendDocument.replyMarkup = keyboard

        bot.execute(sendDocument)
    }

    override fun help(chatId: String): String {
        return bot.resourceOperator.getHelp("plan", chatId)!!
    }

    override fun name(): String {
        return "/plan"
    }
}